modules/API/pages/index.adoc

```diff
@@ -0,0 +1,41 @@
+= GSQL SERVER REST API
+:description: Overview of GSQL SERVER's REST API.
+:pp: {plus}{plus}
+:page-aliases: API:intro.adoc, api:intro.adoc
+
+This user guide provides information on how to engage with our REST APIs: the introduction section explains how to send requests, pass parameters, and format request bodies, while xref:gsql-endpoints.adoc[] describes in detail each endpoint and its input and output.
+
+To submit a request, send an HTTP request to the NGINX server or the GSQL server.
+By default, the GSQL server listens for requests at port 8123 and the NGINX server listens on port 14240.
+A request needs to specify the following:
+
+* The request method (`GET`, `POST`, `PUT`, or `DELETE`)
+* The endpoint address
+* Any required or optional request parameters in URL encoding
+* For `POST` requests, a data payload (i.e., request body) in JSON encoding unless otherwise specified
+* In some cases, request header parameters
+
+For requests that are sent to the GSQL server, the sender needs to provide TigerGraph user credentials for the request to be accepted.
+
+
+== `curl` options
+
+Request examples in this guide are made using https://curl.se/docs/manpage.html[`curl`]. Below is a list of `curl` options used in our code examples:
+
+* `-d <data>`
+ ** Sends the specified data in a `POST` request to the HTTP server in the same way that a browser does when a user has filled in an HTML form and presses the submit button. This will cause curl to pass the data to the server using the content-type `application/x-www-form-urlencoded`.
+ ** If you start the data with the character `@`, the rest should be a filepath from which to read the data. The command `curl -d @foobar` will read data from a file named `foobar`.

```

Comment:
Let's try to avoid "you" and "we".

Change it to: If the data starts with the character @, the rest should specify the filepath from which to read the data.